NumLib/Fem/CoordinatesMapping/NaturalCoordinatesMapping-impl.h

```diff
@@ -120,7 +120,8 @@ inline void computeMappingMatrices(
             shapemat.dNdx.topLeftCorner(ele_dim, nnodes) = shapemat.invJ * shapemat.dNdr;
         } else {
             auto const& matR = ele_local_coord.getRotationMatrixToGlobal(); // 3 x 3
-            auto dshape_global = matR.topLeftCorner(3u, ele_dim) * (shapemat.invJ * shapemat.dNdr); //3 x nnodes
+            auto invJ_dNdr = shapemat.invJ * shapemat.dNdr;
```

Comment:
Can `invJ_dNdr` be `const`? If yes, I am unsure if the `auto`-type is already const?